charts/posthog/templates/test-templates/clickhouse-env.yaml

```diff
@@ -0,0 +1,9 @@
+{{- if .Values.testTemplates -}}
```

Comment:
Can you add a `README.md` in this folder with the explanation on why we need those templates? I think I've linked you recently the issue in helm unittest about it (but I can't find it anymore)

Reply:
I can't - it would break helm templating. :(

I could rename this to `templates/_test_template.yaml` though and add the comment there? We could add all the testing-specific templates to a file instead of a folder then.

Reply:
sounds good but feel free to merge this and then we can iterate further